recipe/recipe.yaml

```diff
@@ -44,9 +48,10 @@ tests:
         - vsg
   - script:
       - vsg --help
-      # Verify correct version of the CLI
-      - vsg --version | grep "${{ version }}"
-      # Verify correct version of the module at runtime
+      - if: not win
+        then: vsg --version | grep "${{ version }}"
+      - if: win
+        then: vsg --version | findstr ${{ version }}
```

Comment:
The python module test below is more portable and should catch any regressions in the version number. Lets just drop this CLI test instead.

Reply:
Ok, I'm going to drop them.